prijateli_tree/app/routers/games.py

```diff
@@ -201,107 +275,44 @@ def route_add_answer(
     db.commit()
     db.refresh(new_answer)
 
-    return {"status": "New answer recorded", "round": current_round}
+    return new_answer
```

Comment:
The post request should lead to the next page. Imagine a user clicks on the "red" button. It will lead to this POST request route and then send them on to the next webpage -- you can do that with `RedirectResponse(url=url, status_code=HTTPStatus.FOUND)`